eve/main.yml

```diff
@@ -243,6 +246,7 @@ models:
       name: Create bootstrap configuration file
       env:
         PRODUCT_VERSION: "%(prop:metalk8s_version)s"
+        DEBUG: "%(prop:metalk8s_debug:-false)s"
```

Comment:
Could be cool if all `env` could inherit from a `base_env` which sets this `DEBUG` variable, but it means changing much more stuff.

Reply:
Yeaaah... Could be cool to have a better test manager than YAML.